tests/tests/test-gas-estimation2.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from "chai";
+
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+
+import { createContract } from "../util/transactions";
+
+describeDevMoonbeam("Estimate Gas - infinite loop", (context) => {
+  it("Should be able to estimate gas of infinite loop call", async function () {
+    this.timeout(60_000);
```

Comment:
Is this expected to take 60 seconds? We probably don't want this running in CI if it is.

Reply:
Sorry, just caught up on the previous conversation. If your test now only takes 23ms, maybe we should remove `this.timeout(60_000)` (or set it to a low value).

Reply:
Done